VRDR.CLI/1.xml

```diff
@@ -594,115 +594,198 @@
+        </valueCodeableConcept>
+        <component>
+          <code>
+            <coding>
+              <system value="http://loinc.org" />
+              <code value="69440-6" />
+              <display value="Disease onset to death interval" />
+            </coding>
+          </code>
+          <valueCodeableConcept>
+            <text value="minutes" />
+          </valueCodeableConcept>
+        </component>
+      </Observation>
     </resource>
   </entry>
   <entry>
     <fullUrl value="urn:uuid:5c0a0509-8297-4a53-a0ef-725daaaf9736" />
     <resource>
-      <Condition>
+      <Observation>
         <id value="5c0a0509-8297-4a53-a0ef-725daaaf9736" />
         <meta>
-          <profile value="http://hl7.org/fhir/us/vrdr/StructureDefinition/VRDR-Cause-Of-Death-Condition" />
+          <profile value="http://hl7.org/fhir/us/vrdr/StructureDefinition/vrdr-cause-of-death-part1" />
         </meta>
-        <code>
+          <code>
           <coding>
-            <system value="http://hl7.org/fhir/sid/icd-10" />
-            <code value="I21.9" />
-            <display value="Acute myocardial infarction, unspecified" />
+            <system value="http://loinc.org" />
+            <code value="69453-9" />
+            <display value="Cause of death [US Standard Certificate of Death]" />
           </coding>
-          <text value="Acute myocardial infarction" />
         </code>
         <subject>
           <reference value="urn:uuid:0042d54e-25fe-426d-8a3e-d002369951d8" />
         </subject>
-        <onsetString value="6 days" />
-        <asserter>
+        <performer>
           <reference value="urn:uuid:c99651c4-7c84-499f-bb37-7db96f9929c4" />
-        </asserter>
-      </Condition>
+        </performer>
+        <valueCodeableConcept>
+          <coding>
+            <system value="http://hl7.org/fhir/sid/icd-10" />
+            <code value="I21.9" />
+            <display value="Acute myocardial infarction, unspecified" />
+          </coding>
+          <text value="Acute myocardial infarction" />
+        </valueCodeableConcept>
+        <component>
+          <code>
+            <coding>
+              <system value="http://loinc.org" />
+              <code value="69440-6" />
+              <display value="Disease onset to death interval" />
+            </coding>
+          </code>
+          <valueCodeableConcept>
+            <text value="6 days" />
+          </valueCodeableConcept>
+        </component>
+      </Observation>
     </resource>
   </entry>
   <entry>
     <fullUrl value="urn:uuid:826a3c27-c712-4c65-b454-46cc4ca76c99" />
     <resource>
-      <Condition>
+      <Observation>
         <id value="826a3c27-c712-4c65-b454-46cc4ca76c99" />
         <meta>
-          <profile value="http://hl7.org/fhir/us/vrdr/StructureDefinition/VRDR-Cause-Of-Death-Condition" />
+          <profile value="http://hl7.org/fhir/us/vrdr/StructureDefinition/vrdr-cause-of-death-part1" />
```

Comment:
So there can be 0...* http://hl7.org/fhir/us/vrdr/StructureDefinition/vrdr-cause-of-death-part1 profiles in a death record? What is the significance of having multiple? Is it data entered at different times?